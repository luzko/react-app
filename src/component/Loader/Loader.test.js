import React from "react";
import renderer from 'react-test-renderer';
import Loader from "./Loader";

test('snapshot test for loader', () => {
  const component = renderer.create(
      <Loader/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
