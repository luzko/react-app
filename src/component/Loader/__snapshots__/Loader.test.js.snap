// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for loader 1`] = `
<div
  className="loader"
/>
`;
